Sources/PointFree/VideoJs.swift

```diff
@@ -10,18 +10,61 @@ var videoJsHead: [ChildOf<Element.Head>] {
       rel(.stylesheet)
       ]),
     .init(script([
+      .init("onload", "videoJsLoaded()"),
       src("https://cdnjs.cloudflare.com/ajax/libs/video.js/7.2.4/video.min.js"),
       `defer`(true)
       ]))
   ]
 
   return [
     style(".vjs-subs-caps-button" % display(.none)),
-    script("window.HELP_IMPROVE_VIDEOJS = false;")
+    .init(node("script", [.text(unsafeUnencodedString("""
+window.HELP_IMPROVE_VIDEOJS = false
+
+function videoJsLoaded() {
+  video.addEventListener('webkitplaybacktargetavailabilitychanged', function(event) {
+    if (event.availability != "available") { return }
+
+    videojs("episode-video").ready(function() {
```

Comment:
and finally another callback to know when video.js has been initialized for our video container.